app/views/histories/10_downing_street.html.erb

```diff
@@ -0,0 +1,307 @@
+      </span>
+      <figcaption>Whitehall from St James's Park &ndash; Hendrick Danckerts c.1675</figcaption>
+    </figure>
+
+    <p class="govuk-body">The earliest building known to have stood on the site of Downing Street was the Axe brewery owned by the Abbey of Abingdon in the Middle Ages. By the early 1500s, it had fallen into disuse.</p>
+    <p class="govuk-body">Henry VIII (reigned 1509 to 1547) developed Westminster's importance further by building an extravagant royal residence there.</p>
+    <p class="govuk-body">Whitehall Palace was created when Henry VIII confiscated York House from Cardinal Wolsey in 1530 and extended the complex. Today's Downing Street is located on the edge of the Palace site.</p>
+    <p class="govuk-body">The huge residence included tennis courts, a tiltyard for jousting, a bowling green, and a cockpit for bird fights. Stretching from St James's Park to the Thames, it was the official residence of Tudor and Stuart monarchs until it was destroyed by fire in 1698. It made the surrounding real estate some of the most important and valuable in London &ndash; and the natural home of power.</p>
+    <p class="govuk-body">The first domestic house known to have been built on the site of Number 10 was a large building leased to Sir Thomas Knyvet in 1581 by Queen Elizabeth I (reigned 1558 to 1603). He was one of the Queen's favourites and was an MP for Thetford as well as a justice of the peace for Westminster. His claim to fame was the arrest of Guy Fawkes for his role in the gunpowder plot of 1605. He was knighted in 1604 by Elizabeth's successor, King James I (reigned 1603 to 1625), and the house was extended.</p>
+    <p class="govuk-body">After the death of Sir Knyvet and his wife, the house passed to their niece, Elizabeth Hampden, who continued to live there for the next 40 years.</p>
+    <p class="govuk-body">The middle of the 17th century was a period of political upheaval and Mrs Hampden's family was right in the middle of it. Her son, John Hampden, was one of the MPs who opposed King Charles I (reigned 1625 to 1649), and Oliver Cromwell, the Lord Protector, was Mrs Hampden's nephew.</p>
+    <p class="govuk-body">Hampden House, as it was then known, gave Mrs Hampden a prime view of the tumultuous events during the Civil War and the Commonwealth and the early years of the Restoration.</p>
+    <p class="govuk-body">The execution of Charles I in 1649 took place on a scaffold in front of Banqueting House in Whitehall, within earshot of the house. Mrs Hampden was still living there when King Charles II (reigned in Scotland from 1649 to 1685) was restored to the English throne in 1660.</p>
+    <p class="govuk-body">The Parliamentary Commissioners, who took over Crown lands during the time of the Commonwealth, described the house in 1650:</p>
+    <blockquote class="app-quote">
```

Comment:
We do have quote styling within the govspeak component, I wonder if we could use that somehow to avoid needing custom styling.